stacks/analytics-ingest-lambda.yml

```diff
@@ -149,6 +175,45 @@ Resources:
       VpcConfig:
         SecurityGroupIds: !Split [",", !FindInMap [VPCSettingsMap, !Ref EnvironmentTypeAbbreviation, vpcsecurity]]
         SubnetIds: !Split [",", !FindInMap [VPCSettingsMap, !Ref EnvironmentTypeAbbreviation, vpcsubnets]]
+  AnalyticsDynamodbLambdaFunction:
+    Type: "AWS::Lambda::Function"
+    Properties:
+      Code:
+        S3Bucket: !Ref CodeS3Bucket
+        S3Key: lambda/PRX-analytics-ingest-lambda.zip
+        S3ObjectVersion: !Ref CodeS3ObjectVersion
+      Description: Dovetail analytics to dynamodb
+      Environment:
+        Variables:
+          DYNAMODB: "true"
+          DDB_TABLE: !Ref DynamodbTableName
+          DDB_ROLE: !Ref DynamodbAccessRole
+          DDB_TTL: "604800"
+          KINESIS_STREAM: !Ref MetricsKinesisStream
+      Handler: index.handler
+      MemorySize: 256
+      Role: !GetAtt AnalyticsLambdaExecutionIAMRole.Arn
+      Runtime: nodejs8.10
+      Tags:
+        - Key: Project
+          Value: Dovetail
+        - Key: Environment
+          Value: !Ref EnvironmentType
+        - Key: "prx:cloudformation:stack-name"
+          Value: !Ref AWS::StackName
+        - Key: "prx:cloudformation:stack-id"
+          Value: !Ref AWS::StackId
+      Timeout: 30
+  # Lambda Triggers
+  # TODO: bigquery/pingbacks/redis triggers manually managed, and tricky to move
+  AnalyticsDynamodbKinesisTrigger:
+    Type: "AWS::Lambda::EventSourceMapping"
+    Properties:
+      BatchSize: 100
```

Comment:
Any benefit of moving this to SSM so it can be updated without a deploy?

Reply:
It would still require a deploy, right?  But not a code-change to Infrastructure.